cfgov/unprocessed/css/molecules/inset.less

```diff
@@ -1,51 +1,3 @@
-/* topdoc
-  name: Insets near Full Width Text
-  family: cfgov-molecules
-  patterns:
-    - name: Default example
-      markup: |
-        <div class="o-full-width-text-group">
-            <div class="m-full-width-text"></div>
-            <div class="m-inset
-                        m-inset__bottom-rule
-                        m-inset__left
-                        m-inset__portrait
-                        m-inset__270">
-                <div class="m-inset_image-container">
-                    <img class="m-inset_image" />
-                </div>
-                <div class="m-inset_caption"></div>
-                <div class="m-quote"></div>
-                <div class="m-related-links"></div>
-                <div class="m-call-to-action"></div>
-            </div>
-            <div class="m-full-width-text"></div>
-        </div>
-      codenotes:
-        - |
-          Structural cheat sheet:
-          -----------------------
-          .o-full-width-text
-            .m-full-width-text
-            .m-inset
-            .m-inset__bottom-rule
-            .m-inset__left
-            .m-inset__portrait
-            .m-inset__270
-                .m-inset_image-container
-                    img.m-inset_image
-                .m-inset_caption
-                .m-quote
```

Comment:
Looks like `m-quote` is unused and could be removed from the CSS.

Reply:
Yep, I have a running list of notes for follow-on PRs, thanks for pointing this out!